daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -258,6 +269,10 @@ private[lf] object Speedy {
     private[speedy] def contractIdByKey(keyHash: crypto.Hash): Option[SValue.SContractId] =
       keyMap.get(keyHash)
 
+    private[speedy] def isValidEntry(key: GlobalKey, contractId: V.ContractId): Boolean = {
```

Comment:
what do you mean by entry ?

Reply:
Renamed to the less generic `isValidDisclosedContractKeyEntry`